addons/l10n_in/models/res_partner.py

```diff
@@ -49,6 +55,8 @@ def onchange_vat(self):
             state_id = self.env['res.country.state'].search([('l10n_in_tin', '=', self.vat[:2])], limit=1)
             if state_id:
                 self.state_id = state_id
+            if self.vat[2].isalpha():
+                self.l10n_in_pan = self.vat[2:12]
```

Comment:
should be done only for india, I suppose :thinking:

Reply:
```python
if self.vat and self.check_vat_in(self.vat):
``` 

already have the `vat` validation for india

Reply:
mmhh'kay, I suppose it'll do